generate/generate.go

```diff
@@ -248,8 +248,8 @@ func (g *Generator) generatePlugin(req plugin.CodeGeneratorRequest, gen config.G
 		gpkg := g.gunkPkgs[pkgPath]
 		data := []byte(*rf.Content)
 		dir := gpkg.Dir
-		if gen.Out != "" {
-			dir = gen.Out
+		if op := gen.OutPath(); op != "" {
```

Comment:
`op` generally means `operator`, so I'd change this variable to `path` or `outPath`.